### PR TITLE
Add internal shipment html report

### PR DESCRIPTION
**Pull request by usudaysingh**

### CODE REVIEW

File: report_html_stock.xml [hunk 1]
```diff
@@ -95,5 +95,21 @@
             <field name="report">report_html_stock/reports/product_ledger.html</field>
             <field name="extension">html</field>
         </record>
+        <record model="ir.action.report" id="stock.report_shipment_internal">
+          <field name="active" eval="False"/>
```
Comment (priyankarani) [inline report_html_stock.xml]: why inactive? just write all other values here

File: report_html_stock.xml [hunk 2]
```diff
@@ -95,5 +95,21 @@
...
+        </record>
+        <!-- Internal Shipment -->
+        <record model="ir.action.report" id="internal_shipment_html">
+            <field name="name">Internal Shipment</field>
+            <field name="model">stock.shipment.internal</field>
+            <field name="report_name">report.internal_shipment</field>
+            <field name="report">report_html_stock/reports/internal_shipment.html</field>
+            <field name="extension">pdf</field>
```
Comment (priyankarani) [inline report_html_stock.xml]: do not create new record. just use stock.report_shipment_internal to add new report path

File: report_html_stock.xml [hunk 3]
```diff
@@ -95,5 +95,21 @@
...
+        </record>
+        <record model="ir.action.keyword" id="internal_shipment_report_keyword">
+            <field name="keyword">form_print</field>
+            <field name="model">stock.shipment.internal,-1</field>
+            <field name="action" ref="internal_shipment_html"/>
+        </record>
```
Comment (priyankarani) [inline report_html_stock.xml]: do not create keyword again

File: report_html_stock.py [hunk 4]
```diff
@@ -179,6 +179,10 @@ class DeliveryNote(ReportMixin):
     __name__ = 'report.delivery_note'
 
 
+class InternalReport(ReportMixin):
```
Comment (priyankarani) [inline report_html_stock.py]: name it InternalShipmentReport

File: reports/internal_shipment.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+      <td align="right"><b>Shipment #</b></td>
+      <td>{{ shipment.code }}</td>
+    </tr>
+    <tr>
+      <td align="right"><b>Date</b></td>
+      <td>{{ shipment.effective_date or datetime.datetime.today()|dateformat }}</td>
+    </tr>
+  </table>
+{% endblock report_header %}
+<h2> Moves </h2>
```
Comment (priyankarani) [inline reports/internal_shipment.html]: moves?

File: report_html_stock.xml (hunk 2 above)
Comment (priyankarani) [inline report_html_stock.xml]: dont put extension

File: reports/internal_shipment.html [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+        <tr>
+          <td class="text-center">{{ shipment.reference or '' }}</td>
+          <td class="text-center">{{ shipment.from_location.rec_name }}</td>
+          <td class="text-center">{{ shipment.to_location.rec_name }}</td>
+          <td class="text-center">{{ shipment.planned_date or ''}}</td>
+          <td class="text-center">{{ shipment.state }}</td>
+        </tr>
+      </tbody>
+    </table>
+
```
Comment (priyankarani) [inline reports/internal_shipment.html]: these must be some heading or something like "Moves" check other report how they do it

Comment (usudaysingh) [inline reports/internal_shipment.html]: Did not find any reference to add heading in between. Can add heading using standard HTML.

**Comment by priyankarani** [comment]
LGTM

**Comment by prakashpp** [comment]
attach a screenshot.

**Comment by usudaysingh** [comment]
thumbnail

**Comment by prakashpp** [comment]
@usudaysingh date format is not correct. Check other reports. Also quantity format is wrong. Please fix.

**Comment by usudaysingh** [comment]
screen shot 2015-12-17 at 7 29 33 pm

**Pull request merged**